packages/cli/src/lib/tasks.ts

```diff
@@ -73,6 +74,10 @@ export async function templatingTask(
     throw new Error(`Failed to read template directory: ${error.message}`);
   });
 
+  const isMonoRepo = (await fs.pathExists(paths.resolveTargetRoot('plugins')))
```

Comment:
This isn't actually true for apps created with `create-app`, even though it's a monorepo. The simplest check for commands that run inside packages is `paths.targetRoot === paths.targetDir`, but since this is supposed to be run at the root I think checking for `paths.resolveTargetRoot(lerna.json)` is the most sensible option.

Reply:
I tried `paths.targetRoot === paths.targetDir` on scaffold plugin outside of monorepo, it doesn't work.

I'll use `paths.resolveTargetRoot(lerna.json)` to detect monorepo.